plugins/modules/digital_ocean_kubernetes_info.py

```diff
@@ -21,35 +21,33 @@
       - DigitalOcean OAuth token; can be specified in C(DO_API_KEY), C(DO_API_TOKEN), or C(DO_OAUTH_TOKEN) environment variables
     type: str
     aliases: ['API_TOKEN']
-    required: yes
+    required: true
   name:
     description:
       - A human-readable name for a Kubernetes cluster.
     type: str
-    required: yes
+    required: true
   return_kubeconfig:
     description:
       - Controls whether or not to return the C(kubeconfig).
     type: bool
-    required: no
-    default: False
-requirements:
-  - python >= 2.6
+    required: true
```

Comment:
This wasn't required before. Is this change intentional?

Reply:
Nope, not required, currently fixed (defaulted to `false`).